flink-clients/src/main/java/org/apache/flink/client/cli/CheckpointOptions.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.client.cli;
+
+import org.apache.flink.core.execution.CheckpointType;
+
+import org.apache.commons.cli.CommandLine;
+
+import static org.apache.flink.client.cli.CliFrontendParser.CHECKPOINT_FULL_OPTION;
+
+/** Command line options for the CHECKPOINT command. */
+public class CheckpointOptions extends CommandLineOptions {
+
+    private final String[] args;
+
+    private CheckpointType checkpointType;
+
+    public CheckpointOptions(CommandLine line) throws CliArgsException {
+        super(line);
+        args = line.getArgs();
+        // TODO: Support type INCREMENTAL
```

Comment:
would it be better? 

```suggestion
        // TODO: [FLINK-33498] Support type INCREMENTAL
```

Reply:
Hi @JingGe , thanks for the reminder.

I'm afraid the FLINK-33498 is introduced to disabling INCREMENTAL everywhere and throw exception if tried. Supporting manually triggered incremental cp is a bigger story which needs much discussion and work involving several tickets, so I suggest not putting any specific ticket here.